cypress/integration/unit/search.spec.js

```diff
@@ -156,6 +156,27 @@ context('Search tests', () => {
 
       cy.get('[data-table]').contains('korte titel for batterij');
     });
+
+    cy.get('[data-test-m-header-settings]').click();
+    cy.wait(1000);
+    cy.get('[data-test-m-header-search]').click();
+    cy.wait(1000);
+    cy.get('[data-test-searchfield]').should('have.value', '');
+  });
+
+  it('Searchfield should be empty after revisting search page', () => {
+    cy.visit('/zoeken/agendapunten');
+    cy.get('[data-test-searchfield]').clear();
+    cy.get('[data-test-searchfield]').type('TestSearchSet');
+    cy.server();
+    cy.route('GET', '/agendaitems/search?**').as('searchCall');
```

Comment:
We schrijven in kaleidos onze selectors in een aparte file. Check bv de `agenda.selectors.js` en `propagatie-overheid.spec.js` daar zie je hoe we het hier geïmplementeerd hebben.